FileOperations/FileOperations/FileDescription.cs

```diff
@@ -9,15 +9,15 @@ namespace FileOperations
 {
 	class FileDescription
 	{
-        public const string Path = "C:/Users/divya.singh/Desktop/demo/testFile.txt";
+        public const string path = "./Data/Hello1.txt";
 
-		public void createFileAndAddText()
+		public void readText()
```

Comment:
The problem statement talks about "creating a file and adding text to it". Which means, creating a blank file and add text to it. Not just to read text.